resources/replicaset.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2016 Mirantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resources
+
+import (
+	"errors"
+	"log"
+	"strconv"
+
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+type ReplicaSet struct {
+	ReplicaSet *extensions.ReplicaSet
+	Client     unversioned.ReplicaSetInterface
+}
+
+func getSuccessFactor(meta map[string]string) (int32, error) {
+	var successFactor string
+	var ok bool
+	if meta == nil {
+		successFactor = "100"
+	} else if successFactor, ok = meta["success_factor"]; !ok {
+		successFactor = "100"
+	}
+
+	sf, err := strconv.ParseInt(successFactor, 10, 32)
```

Comment:
Yes, i know, but `Status.Replicas` and `Spec.Replicas` are int32 based.
So yes, we can use there `Atoi` and plain `int` above, then cast it to `int32` in line 56.